media/css/firefox/facebook-container.scss

```diff
@@ -2,21 +2,51 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-$font-path: '/media/fonts';
-$image-path: '/media/protocol/img';
+$font-path: "/media/fonts";
+$image-path: "/media/protocol/img";
 
 @import "../../protocol/css/includes/lib";
-@import '../../protocol/css/components/hero';
-@import '../../protocol/css/templates/card-layout';
+@import "../../protocol/css/components/hero";
+@import "../../protocol/css/components/split";
+@import "../../protocol/css/components/logos/logo";
+@import "../../protocol/css/components/logos/logo-product-firefox";
+@import "../../protocol/css/templates/card-layout";
 @import "../../protocol/css/components/call-out";
-@import '../../protocol/css/components/feature-card';
+@import "../../protocol/css/components/feature-card";
 
 //* -------------------------------------------------------------------------- */
 // Components
-.mzp-c-hero.mzp-t-dark {
+.mzp-c-split-bg.mzp-t-dark {
     background-color: $color-ink-80;
     color: $color-white;
-    position: relative;
+
+    .mzp-c-split-container {
+        box-sizing: content-box;
+
+        .mzp-c-split-body {
+            display: flex;
+            justify-content: center;
+            flex-direction: column;
+            max-width: 580px;
+            .mzp-c-logo {
+                align-self: center;
+                @media #{$mq-md} {
+                    align-self: inherit;
+                }
+            }
+            .mzp-u-title-xl {
+                padding-top: $spacing-md;
+                margin-bottom: $spacing-md;
+                @media #{$mq-md} {
+                    padding-top: $spacing-lg;
+                    margin-bottom: $spacing-xl;
+                }
+            }
+            .mzp-u-title-3xs {
+                @include text-title-3xs;
```

Comment:
@craigcook when i looked here: https://github.com/mozilla/protocol/blob/main/src/assets/sass/protocol/base/utilities/_titles.scss the 3xs class wasn't being declared. Is it somewhere else?

Reply:
Huh, you're right, seems we don't have that class. I think we added the 3xs size *after* we created the utility classes and neglected to add a class for the new size.

So in that case, maybe we should omit the `mzp-` prefix here since this isn't actually coming from Protocol (and we should add that utility class upstream).